src/components/BottomSheet/BottomSheet.js

```diff
@@ -39,14 +39,16 @@ const styles = StyleSheet.create({
   container: {
     paddingTop: dimen.normalizeDimen(10),
     paddingHorizontal: dimen.normalizeDimen(20),
-    paddingBottom: dimen.normalizeDimen(20)
+    paddingBottom: dimen.normalizeDimen(20),
+    backgroundColor: COLORS.almostBlack
   },
   containerSheet: (pullBottom) => ({
     borderTopRightRadius: 20,
     borderTopLeftRadius: 20,
-    justifyContent: pullBottom ? 'flex-end' : 'flex-start'
+    justifyContent: pullBottom ? 'flex-end' : 'flex-start',
+    backgroundColor: COLORS.almostBlack
   }),
   draggableIcon: {
-    backgroundColor: COLORS.lightgrey
+    backgroundColor: COLORS.gray100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `draggableIcon`'s background color from `COLORS.lightgrey` to `COLORS.gray100` seems fine, assuming `COLORS.gray100` is a valid and defined color in your project's color scheme. Please make sure this color change doesn't affect the visibility of the icon against its new background color.


<!-- This is an auto-generated comment by OSS CodeRabbit -->